GruntFile.js

```diff
@@ -132,7 +132,8 @@ module.exports = function(grunt) {
                         'bower_components/d3/d3.js',
                         'src/*.js',
                         'src/models/*.js',
-                        'test/mocha/*.coffee'
+                        'test/mocha/*.coffee',
+	                    'https://cdn.rawgit.com/Kcnarf/d3-beeswarm/fbda9b54/build/d3-beeswarm.min.js',
```

Comment:
It's a small library that positions observations in a "swarm" pattern:
<img width="339" alt="screen shot 2017-08-24 at 09 20 27" src="https://user-images.githubusercontent.com/8518288/29670824-8140f6b4-88ad-11e7-820a-4a8f30f14ec8.png">

I hear ya on the external dependency, would it be better to simply include the source code in this chart type?